src/algebra/module/linear_map.lean

```diff
@@ -794,6 +794,10 @@ instance apply_smul_comm_class : smul_comm_class R (module.End R M) M :=
 instance apply_smul_comm_class' : smul_comm_class (module.End R M) R M :=
 { smul_comm := linear_map.map_smul }
 
+instance _root_.module.End.is_scalar_tower
```

Comment:
```suggestion
instance apply_is_scalar_tower
```
Probably this whole section should be moved into the `module.End` namespace, but that's only in scope for this PR if you want it to be.

Reply:
I'll descope the renamespacing for now but add it to my list of things tidy up.

In the meantime, I agree this name is much more consistent.